examples/dmd/local_dw_csv.cpp

```diff
@@ -0,0 +1,889 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Compile with: make local_dw_csv
+//
+// Generate CSV databse on heat conduction with: heat_conduction_csv.sh
+//
+// =================================================================================
+//
+// Local serial DMD command:
```

Comment:
Doesn't "local" mean for a single window out of multiple, so that "serial" means global? If so, shouldn't it be just `Serial DMD command`? The same goes for `local_tw_csv.cpp`.

Reply:
The word "local" in the scripts `local_*w_csv` means a single sequence of DMD models from one training point. The sequence consists of N >= 1 windows, where N = 1 means serial. The prediction will be done by directly applying the sequence of N DMD models. On the contrary, "parametric" in the scripts `parametric_*w_csv` means multiple sequences of DMD models from M training points, which have altogether M*N DMD models. The prediction will be done by interpolating the sequence of N DMD models with respect to the parameter points.